### ISSUE TITLE
Angular Material documentation code snippets refer to the old md-* prefixes

### ISSUE THREAD

**Issue by Bartosz-D3V** [angular/material2#8175]
#### Bug, feature request, or proposal:
Bug (in documentation)

#### What is the expected behavior?
Documentation should reflect the change from md prefix to mat prefix in material components

#### What is the current behavior?
Documentation for the latest version of Angular Material still refers to the old prefix which causes bugs in production.

#### What are the steps to reproduce?
Please take a look at code snippets at Angular Material documentation.
For example:
https://material.angular.io/components/menu/overview still refers to md prefix - similarly to https://material.angular.io/components/select/overview

Basically, all of the code snippets need to be updated.

#### What is the use-case or motivation for changing an existing behavior?
Code snippets should provide reliable examples that could be used in a production.

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular Material 2.0.0-beta.12

#### Is there anything else we should know?
You make awesome work :)

**Comment by willshowell** [comment]
For whatever reason, the old docs are being cached really aggressively. If you clear your cache, you should see the updates (also be sure you have no other tabs open to the docs site). I _think_ this should be fixed during the next release.

**Comment by jelbourn** [comment]
Yeah, this is a caching issue that's service-worker related. Should be resolved soon. Closing this since the docs actually _are_ correct, just cached.
